include/envoy/runtime/runtime.h

```diff
@@ -22,6 +22,13 @@ class RandomGenerator {
   virtual uint64_t random() PURE;
 
   /**
+   * Writes out_len bytes of random data to out.
+   * @param out supplies output buffer.
+   * @param out_len supplies number of requested bytes / length of the output buffer.
+   */
+  virtual void random_bytes(uint8_t* out, size_t out_len) PURE;
```

Comment:
nit: I would not bother adding this to the interface unless it is used externally. Would remove for now.